setup.py

```diff
@@ -10,7 +10,7 @@
     keywords='openstreetmap gtfs schedule public-transportation python',
     author='Various collaborators: https://github.com/grote/osm2gtfs/graphs/contributors',
 
-    install_requires=['attrs>=17.1.0', 'overpy>=0.4', 'transitfeed>=1.2.16', 'mock', 'webcolors', 'transporthours'],
+    install_requires=['attrs>=17.1.0', 'overpy>=0.4', 'transitfeed @ git+https://github.com/pecalleja/transitfeed.git@python3', 'mock', 'webcolors', 'transporthours'],
```

Comment:
As https://github.com/google/transitfeed is no longer actively maintained, it's probably okay to use such a fork. Wondering what are the alternatives on the long run, because the recommended alternative https://github.com/MobilityData/gtfs-validator only does validation, not parsing for Python, right?